uint32 identification_time_ms
---
bool success
float32 sampling_time
float32[100] samples